edxval/migrations/0004_auto__chg_field_profile_width__chg_field_profile_extension__chg_field_.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    def forwards(self, orm):
+        # The extension, width, and height fields of the Profile model are
+        # removed, but we want this migration to be non-destructive. Thus, the
+        # columns are simply altered to allow null values.
```

Comment:
Actually, this reads strangely as a docstring to me. I don't think it's necessary to include a docstring for a migration function, since they are never really called except by the migration tooling.